changelog.d/20230622_141438_chris_endpoint_setup.rst

```diff
@@ -0,0 +1,9 @@
+.. A new scriv changelog fragment.
+..
+.. Uncomment the header that is right (remove the leading dots).
+..
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Added ``endpoint_setup`` option to endpoint config, which, if present, is run by the
```

Comment:
I think we usually put the '--' to make it more clear, ie.

- Added ``--endpoint_setup`` option to endpoint config, which, if present, is run by the

Reply:
Also, may be better to give a simple example of usage, especially if multi-line commands are supported.

Reply:
Yes that's true.  I thought it was a CLI option when I made the comment.